arbitrum-docs/stylus/stylus-gentle-introduction.md

```diff
@@ -9,123 +9,34 @@ sidebar_position: 1
 
-If you're a developer interested in Stylus, visit the [quickstart](https://docs.arbitrum.io/stylus/stylus-quickstart), join the [Discord channel](https://discord.com/invite/arbitrum), and start building!
+### Use Cases
 
-### Wen mainnet?
+While many developers will be drawn to new use cases, rebuilding existing applications in Stylus will also open the door to innovation and optimization. dApps have never been faster, cheaper, or safer. Stylus can be easily integrated into existing Solidity projects by calling a Stylus contract to optimize specific parts of your dApp or building the entire dApp with Stylus.It's impossible to list all of the use cases Stylus enables, think about the properties of all WASM compatible languages! That said, here are some ideas that are particularly exciting:
 
-Stylus is now live on mainnet.
+- <b>Efficient On-Chain Verification with ZK-Proofs</b>: Enable cost-effective onchain verification using zero-knowledge proving systems for privacy, interoperability, and more (see [case study](https://blog.arbitrum.io/renegade-stylus-case-study/)).
+- <b>Advanced DeFi Instruments</b>:  Power complex financial instruments and processes like synthetic assets, options, and futures with onchain computation via extending current protocols (ie. Uniswap V4 hooks) or building your own.
+- <b>High-Performance On-Chain Logic</b>: Support memory and compute-intensive applications like onchain games and generative art either by writing all of the application in Stylus or enhance performance of existing Solidity contracts by optimizing specific parts.
 
-To keep up with the latest announcements and updates about Stylus:
+### Getting Started
+1. Utilize our [quickstart](https://docs.arbitrum.io/stylus/stylus-quickstart), [examples](https://docs.arbitrum.io/stylus/stylus-quickstart), and [tutorials](https://docs.arbitrum.io/stylus/stylus-quickstart) to help you start building.
+2. Join our Stylus Developer [Telegram](t.me/arbitrum_stylus) group and [Arbitrum Discord](https://discord.gg/arbitrum) for support as well as the official Arbitrum ([@Arbitrum](https://twitter.com/arbitrum)) and Arbitrum Developers ([@ArbitrumDevs](https://twitter.com/ArbitrumDevs)) X accounts for announcements.
```

Comment:
Absolutely links like this need to be like https://t.me/arbitrum_stylus

Reply:
That's why the vercel preview failed

Reply:
ty! fixed